include/RAJA/pattern/params/forall.hpp

```diff
@@ -15,6 +15,13 @@
 
 namespace RAJA
 {
+
+inline bool RAJA_caliper_profile = true;
```

Comment:
@LLNL/raja-core the inline call requires c++17, we do plan to update to 20 moving forward right?

Reply:
Yes!